scripts/new-package.js

```diff
@@ -0,0 +1,52 @@
+// a script that takes a string as value and copies the folder packages/template
+// to a new folder with the name of the value
+
+// Include process module
+const { exit } = require("process");
+const process = require("process");
+
+if (process.argv.length === 3) {
+  var projectName = process.argv[2];
+
+  // do some quick sanitization
+  var cleanProjectName = projectName.replace(/[|&;$%@"<>()+,]/g, "");
+
+  // copy the package/template folder to a new folder with the name of the value
+  // and then change the package.json file to the new name
+  const fse = require("fs-extra");
+  const fs = require("fs");
```

Comment:
Hmm… According to their README, we should be able to do:
```suggestion
  const fs = require("fs-extra");
```
Does this work?

Reply:
That's what I did originally, but fs-extra's `writeFileSync` was erroring out 😅 so I decided to only rely on it for the `copySync` method.